exercises/resistor-color-duo/resistor-color-duo.ts

```diff
@@ -0,0 +1,6 @@
+
+
+
+export const value = (_: string[]): number => {
```

Comment:
Are they other ways to solve this?

Reply:
Please throw an error.

Reply:
Wrapped the value method in a class and updated example code to throw error and is covered via test.